includes/class-content-toc.php

```diff
@@ -134,10 +135,12 @@ public function shortcode( $atts, $content = null ) {
 
 			$toc_html .= apply_filters(
 				'hm_content_toc_list',
-				'<ul id="hm-content-toc-' . esc_attr( $this->id_counter ) . '" class="hm-content-toc">' . $toc_list . '</ul>',
+				'<ul class="hm-content-toc">' . $toc_list . '</ul>',
```

Comment:
@sanchothefat do I need to include the wrapper HTML here to, so everything can be changed in the filter?